forge-gui/res/cardsfolder/upcoming/trash_the_town.txt

```diff
@@ -0,0 +1,13 @@
+Name:Trash the Town
+ManaCost:G
+Types:Instant
+K:Spree
+A:SP$ Charm | Choices$ DBCounters,DBTrample,DBDrawTwo | MinCharmNum$ 1 | CharmNum$ 3 | Spree$ True
+SVar:DBCounters:DB$ PutCounter | SpreeCost$ 2 | ValidTgts$ Creature | TgtPrompt$ Select target creature | CounterType$ P1P1 | CounterNum$ 2 | SpellDescription$ Put two +1/+1 counters on target creature.
```

Comment:
these kind of simple TgtPrompts shouldn't be needed

Reply:
Sure enough they aren't. Will keep that in mind going forward.